### ISSUE TITLE
Unit 2: exercise 2: Accessing the Watson Language Translator service from the Node.js app

### ISSUE THREAD

**Issue by gemin55** [NorhanKhaled/SkillsAcademy-CAD#91]
Hi 
Unit 2 Exercise 2
For Part 5 and 6 of this exercise, could you please create a simple web interface (with two textboxes) to access the translator service rather than hard coding the conversion.
Will be really helpful.

**Comment by ckabuya** [comment]
Definitely, I think this can be done by placing an html document within your project.
